Changes/adds engie contractor alt-titles

LorenLuke opened the pull request:
Changes/adds engie contractor alt-titles to add more varied/specialised occupations.

<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->

CountAlex committed:
Adds away mission map - Mobius Rift.

AtomicWorm committed:
Updates Vat-grown Human sprites

babydoll committed:
Fix runtimes walking on alien plating and busted tanks

babydoll committed:
Make some changes to hydrobase away mission

Added more loot and tools to deal with enemies within
Added a new wing for experimental (random) plants
Reduced goat damage potential and removed one of them
Added unique emp_act to farmdrones
Many small tweaks

Commit:
Adds Crime Scene Analysts

Adds Crime Scene Analysts

Revert "SolGov Rep Badge"

This reverts commit 69886f15909351bb32c46f403f4a4930172d3883.

Merge pull request #19946 from sabiram/revert-19914-merrychristmas

Revert "Add a Christmas tree to the galley"
Merge branch 'dev' into feartheblackout/rideemcowboy
Merge remote-tracking branch 'origin/feartheblackout/rideemcowboy' into feartheblackout/rideemcowboy

Revert "Adds Crime Scene Analysts"

This reverts commit d032431adf5ec85afcec7b36492befdb13adeb6a.

Commit:
Removes Ensigns (and Marine 2nd LTs) from Command positions

comma committed:
Antag headsets no longer intercept everything

It's just too damn powerful to be handed out for free IMO.
An interceptor radio was added to the uplink. It's a normal size item (doesn't fit in pockets).
Syndicate headset key price dropped to minimum because it's barely useless now

BlueNexus committed:
Adds the Shage language

Kilakk committed:
Fixes #14645 and related issues with force-feeding from reagent containers

* check_has_mouth() defined for carbons and defaults to 1
* check_mouth_coverage() moved from humans to carbons and defaults to null
* standard_feed_mob() changed...
** to only allow feeding mobs (incl. yourself) of type living/carbon

LorenLuke committed:
Changes engineering contractor alt-titles from previous few to reflect more varied specialised occupations.

PurpleMartinJCK commented:
Why so many alt titles?

SparklySheep commented:
I'm okay with engineering contractor title but big no on adding more overly specific engineering titles. Engineering is not so busy that you need someone who refuses to do work because it's not their specialty.

comma commented on maps/torch/job/jobs.dm:
```diff
@@ -514,9 +514,14 @@
+		"Mechanical Systems Contractor",
+		"Electrical Systems Specialist",
+		"Information Systems Technician",
+		"EVA Operations Specialist",
```
doing EVA in 26th century with at least 3 centuries of routine space travel isn't worth putting on your resume as your main skillset.

LorenLuke replied:
https://github.com/LorenLuke/Baystation12/blob/862c5383278405434144c6babe3d95e64c8ce5ce/maps/torch/job/jobs.dm#L471

comma commented on maps/torch/job/jobs.dm:
```diff
@@ -514,9 +514,14 @@
-		"Supermatter Specialist",
-		"Information Systems Technician")
+		"Maintenance Assistant",
+		"Mechanical Systems Contractor",
```
What mechanical systems?

LorenLuke replied:
Intended to replace 'Mechanic'/add 'construction' capability as specialisation. I'm not sure why we had the 'Mechanic' in the first place, unless the Petrov needed a new phoron filter and its engines rotated.

LorenLuke commented:
>Why so many?

https://github.com/LorenLuke/Baystation12/blob/862c5383278405434144c6babe3d95e64c8ce5ce/maps/torch/job/jobs.dm#L459-L474

One to match each of the engineering Alt Titles, added one for each engine type (considering that contractors are supposed to be specialised, rather than just 'engine technician'), removed 'Mechanic', kept 'Information Systems Tech'.
So it's basically keeping the few that were there, and just allowing for similar specialties that regular engineers have.

SparklySheep commented:
We should remove those engineering alt-titles too.

Casper3667 commented:
Life support? So.. Engie atmos tech? I like it.

Zero-Bits commented:
I like the main title change. I don't think we need that many alt-titles though. Heck, I'd say if anything, we should have fewer. That said, I'm not against this in any way. Alt-titles don't hurt me.

LorenLuke committed:
Renames HUD icon. Hopefully fixes travis.

EcklesFire commented:
I would rather we don't have a fusion systems technician, please roll it into a general Engine's technician or something.

Mainly because 
1. Whether the RUST is run or not is heavily dependent on whoever's in charge of engineering at the time.
2. People sitting solo on the fore of deck 2 not doing other work for something that only needs messing with for ten minutes is not conductive.

LorenLuke commented:
Confirming code passes travis with new commit. 
Afterwards, will be changing the following:
+Mechanical Systems Contractor -> Structural Integrity Specialist
+Fusion Systems Technician /Supermatter Specialist -> Reactor Technician
+~~EVA Operations Specialist~~

Sunbeamstress committed:
Adds a special formatting function to help process physical custom emotes.

LorenLuke commented:
Travis passed, pushing above.

LorenLuke committed:
Incorporates Suggestions

Fixes merge conflict (I think).

Sunbeamstress committed:
Undatumified (that's a word) the proc, also removed that commented code

Chiquiii committed:
First Aid

PsiOmegaDelta committed:
Automatic changelog update [ci skip]

Zero-Bits committed:
Makes Hydrogen Filterable and Properly Mappable

Part 2

Flamethrower Fix

Fixes

More tweaks and fixes

SM Monitor

minor fix

Tweak

mapmerj

Duller Sprites

Sprites 2, Darkness Boogaloo

Flammenwerfixed

Fixes

mapmerge

woops

atlantiscze commented:
Just a thing to think about - given people often mistake IST's job to be telecommunications (which leads to them begging for telecomms access all the time, despite the fact they shouldn't be there), it might be a good idea to rename the role a bit. Something more generic as "Computer systems specialist", or just condense them into "Electrical systems specialist" role altogether?

LorenLuke commented:
@atlantiscze 
>IT vs Electrics

My intent was to have electrical systems be very much an electrician, dealing with power nets, power distribution, etc. As for the IST, the jury is out in my view, given that such is something to be curbed.

babydoll committed:
Add command hardsuit modules

CountAlex committed:
Adds shield generators monitoring program.

Sunbeamstress committed:
Far less goofy way of auto-capitalizing the pretext.

PsiOmegaDelta committed:
Allow setting (almost) all prefix keys

Players can now customize all prefix keys with the exception of radio channel selection.
This will be made possible later, when radio channels are turned into proper datums.

Prefix keys are the keys which activate:
Visible emotes
Audible emotes
Custom emotes
Radio channel selection
Language selection

babydoll committed:
Map HCMs, remove from lockers, icon tweaks

PsiOmegaDelta committed:
Reduce loadout snowflake

Tajara veils are now subtypes of the appropriate /datum/gear/eyes types.
As such they no longer need to override cost, allowed_roles or other shared values.

PsiOmegaDelta committed:
Adjust some QDEL hints

Some QDEL_HINT_HARDDEL_NOW are now merely QDEL_HINT_HARDDEL

PsiOmegaDelta committed:
Automatic changelog update [ci skip]

Kilakk committed:
Fixes #19965

* The runtime in _internal.dm seemed to concern some adminbus and gib()...
* organ-related variable have been moved from /human to /carbon
** testing shows this does not seem to affect human organ operation
** it should not affect other carbons (brain, nymph) as they do not have organs

Kilakk committed:
Fixes #19948 related to stasis clamp adminbus

* opening a stasis clamp will no longer be possible if it isn't attached
** to a pipe of some kind

Kilakk committed:
Tentatively fixes #19934, can't read null.buildables

* checks for connected_god.form before proceeding
* i'm not sure if there's some underlying issue here,
** or what happened to have this runtime occur to begin with?

Kilakk committed:
Fixes #19905, attempts to handcuff non-carbon mobs

* only carbons can be handcuffed
* added a check for mobs that're already cuffed
* added a ..() return at the end

Sbotkin commented:
Can you add Atmospheric Technician as Engineer's alt-title? Also, how about to remove IST or make it Roboticist's alt-title? Fits more, I guess.

Hubblenaut commented:
Engineering Contractor is just a catch-all for all the rest and not saying anything about their actual qualifications. I'd leave it as maint assistant.

Kilakk committed:
Updates to runtime fixes: handcuffs, organs

* put ..() in else block in handcuffs.dm
* moved organ-related vars to carbon_defines.dm

PsiOmegaDelta committed:
Automatic changelog generation for PR #20044 [ci skip]

AtomicWorm committed:
Slightly changes UBAC mob icons

PsiOmegaDelta committed:
Automatic changelog generation for PR #20041 [ci skip]

PsiOmegaDelta committed:
Automatic changelog update [ci skip]

babydoll committed:
Fix deck 1 airlock not cycling, minor hydrobase stuff

Also remove space's light switch

LorenLuke commented:
@Sbotkin 
>Atmos Tech

https://github.com/LorenLuke/Baystation12/blob/84ffaa31e238c62ff07aadec1254759a603e5962/maps/torch/job/jobs.dm#L473

Seek and ye shall find.

@Hubblenaut 
>catch-all

Except that they're not, strictly speaking, mere assistants. They're specialists in their varying fields and are far above being 'hey, pass me that wrench, assistant.'

babydoll committed:
Make items dried in the drying rack brownish

instead of gray

comma committed:
Tweak to health analyzer output

Adds another stage to brain activity - 'fading', as I felt that 'extremely weak' was too fucking long - it covered 2x the brain damage steps.
Makes message about shock more clear about what it means - at least this monkey was confused by it thinking it means issues with blood/heart etc.

comma committed:
Replaces colorable labcoat with just a coat

Totally different thing and not a labcoat with darker default color.
Makes it available to all formal roles, e.g. suited people.

Casper3667 committed:
fixes some forensic access to records (#20082)

* fixes some forensic access to records

PsiOmegaDelta committed:
Automatic changelog generation for PR #20082 [ci skip]

LorenLuke commented:
Aaaaaaa, how do I git? (If you ask me to squash, I have no idea how...)

LorenLuke commented:
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
